platform/default/sqlite_cache.cpp

```diff
@@ -118,9 +118,9 @@ void SQLiteCache::Impl::get(const Resource &resource, Callback callback) {
                 // Status codes > 1 indicate an error
                 response->error = std::make_unique<Response::Error>(Response::Error::Reason(status));
             }
-            response->modified = getStmt->get<int64_t>(1);
+            response->modified = getStmt->get<time_t>(1);
```

Comment:
I don't think we should change away from `int64_t`. `time_t` is a platform-specific typedef and is a 32 bit value on some systems, which will make us susceptible to the year 2038 problem. If it's defined to something else entirely, compilation may fail because `.get<time_t>()` may not be instantiated.